low-level/framework/base/sspl_constants.py

```diff
@@ -36,6 +36,7 @@
 RESOURCE_PATH = "/opt/seagate/%s/sspl/low-level/json_msgs/schemas/" % (PRODUCT_FAMILY)
 CLI_RESOURCE_PATH = "/opt/seagate/%s/sspl/low-level/tests/manual" % (PRODUCT_FAMILY)
 DATA_PATH = "/var/%s/sspl/data/" % (PRODUCT_FAMILY)
+SB_LOG_FILE_PATH = "/var/log/%s/sspl/" % (PRODUCT_FAMILY)
```

Comment:
We can initialize this with sspl log dir value directly, where for sspl its initialized? Use that key in value here

Reply:
Done the changes